component/web/oauth-common/src/main/java/org/gatein/security/oauth/openid/OpenIdProcessorImpl.java

```diff
@@ -428,8 +428,12 @@ protected URL getUserInfoURL() throws IOException {
 
   @Override
   public void start() {
-    if (StringUtils.isBlank(this.wellKnownConfigurationUrl)) {
-      log.info("wellKnownConfigurationUrl is not configured");
+    boolean openIdEnabled = Boolean.parseBoolean(System.getProperty("exo.oauth.openid.enabled"));
```

Comment:
What is the property is not defined ? Does Boolean.parseBoolean(null) works ?

Reply:
it returns false